src/mocks/jest-server.js

```diff
@@ -0,0 +1,6 @@
+import { setupServer } from 'msw/node';
+import handlers from './handlers.js';
+
+const jestServer = setupServer(...handlers);
```

Comment:
Let's set up the server to fail tests if there is any unhandled server request -- we don't actually want to ping `api.com`

Reply:
@juanca I'm confused, what do you mean? Are you saying we don't want any components to actually hit those endpoints in production by accident?

If so, I'm not sure how we'd test for that here, since in test land we _want_ to hit this endpoint. It's like... the opposite of a circular dependency? 🤣